pallets/delegation/src/lib.rs

```diff
@@ -324,13 +324,13 @@ pub mod pallet {
 
 			let consumed_weight: Weight = if !root.revoked {
 				// Recursively revoke all children
-				let (remaining_revocations, post_weight) = Self::revoke_children(&root_id, &invoker, max_children)?;
+				let (_, post_weight) = Self::revoke_children(&root_id, &invoker, max_children)?;
+
+				// If we didn't return an ExceededRevocationBounds error, we can revoke the root
+				// to.
```

Comment:
too?

Reply:
that's what William said two.

Reply:
∫_{0}^{2π}cosxdx

Reply:
They don't render equations? That's sad.